src/gui/guiChatConsole.cpp

```diff
@@ -639,3 +679,109 @@ void GUIChatConsole::setVisible(bool visible)
 	}
 }
 
+// Return how many "ctrl" keycodes successfully found in string, or 0 on fail
+int GUIChatConsole::setupChatClickCtrlKeys(std::string inputline)
+{
+	m_cache_chat_weblink_ctrl_keys.clear();
+
+	irr::EKEY_CODE kc;
+	std::string stemp;
+	size_t startpos = 0, endpos = 0;
+	while(startpos < inputline.size())
+	{
+		// Foreach delimited string,
+		endpos = inputline.find(',', startpos);
```

Comment:
Also no need to re-invent the wheel here. Iterate though the following:
`std::vector<std::string> parts = split(inputline, ',');`
and `trim()` on each element to ensure whitespaces are gone.